include/swift/Option/Options.td

```diff
@@ -643,6 +643,12 @@ def project_name : Separate<["-"], "project-name">,
 def module_name_EQ : Joined<["-"], "module-name=">, Flags<[FrontendOption]>,
   Alias<module_name>;
 
+def module_path : Separate<["-"], "module-path">,
```

Comment:
Should we call this something like `-debug-module-path`? Even though it's just a frontend flag I think someone seeing it could easily assume it's a more general purpose option

Reply:
Yes, I think that's better. That would also be in line with what I chose for the IRGenOption.